Project.toml

```diff
@@ -2,6 +2,13 @@ name = "LibArchive"
 uuid = "1fe3a451-598f-5901-851b-34c9e7f9ea9a"
 version = "0.9.0"
 
+[deps]
+LibArchive_jll = "1e303b3e-d4db-56ce-88c4-91e52606a1a8"
+
+[compat]
+julia = "1"
```

Comment:
Just note that this actually requires julia 1.3

Reply:
Fixed. https://github.com/yuyichao/LibArchive.jl/commit/8c0fc65f532ed8ba4a9ef9c6dec604a3d2009d70